src/application_config/app_config.h

```diff
@@ -1,8 +1,37 @@
+/**
+ * @file app_config.h
+ * @author Otso Jousimaa <otso@ojousima.net>
+ * @date 2020-04-16
+ * @brief Master configuration base
+ *       Variables which can be adjusted by application_mode_xxxxx.h are preceeded by #ifdef
+ *       2021-12-10 DG12 correct _FLASH_PAGES from 16 to 0x15000/4096-1 see loader configuration.
+ *       2021-11-22 DG12 add comments and rearrange bringing most likely overided items to the top.
+ *       2021-10-01 Otso add dataformats for encrypted  08, FA
+ *
+ * Types of configuration items:
+ *  unconfigurable constatnts that should be moved elsewhere example:
+ *                          LONG_PRESS_TIME,FLASH_*_FILE , FLASH_*_RECORD, _SENSOR_FILE, _SENSOR_ENVI_RECORD ..
+ *  define the hardware present and
+ *   Currently they are compile time defines which can be defined in _mode_xxx
+ * ToDo:
+ *
+ *   SHOULD BE HERE BUT ARE IN app_comms.c // ?  really should be in flash tunable
```

Comment:
e.g. adv_send_count is calculated from configuration. It is in the app_comms header since other parts of program do not need the information and it should not be configured by user.